src/GogoKit/Exceptions/ResourceNotFoundException.cs

```diff
@@ -5,7 +5,7 @@ namespace GogoKit.Exceptions
 {
     public class ResourceNotFoundException : ApiErrorException
     {
-        public ResourceNotFoundException(IApiResponse<ApiError> response) : base(response)
+        public ResourceNotFoundException(IApiResponse<ApiError> response, ApiError apiError) : base(response, apiError)
```

Comment:
Doesn't make sense for this constructor to take an ApiError object...can just pass `null` into the base constructor